tests/unit/test_connection.py

```diff
@@ -189,15 +189,14 @@ def test_use_requested_compression(self, *args):
         c = self.make_connection()
         c._requests = {0: (c._handle_options_response, ProtocolHandler.decode_message, [])}
         c.defunct = Mock()
-        # request snappy compression
-        c.compression = "snappy"
+        # request LZ4 compression
+        c.compression = "lz4"
```

Comment:
Why ?

Reply:
Due to this - https://github.com/scylladb/python-driver/pull/492#issuecomment-2988397021

Reply:
Done, in a separate, additional commit.